doc/nrf/ug_zigbee_qsg.rst

```diff
@@ -199,6 +203,16 @@ To program the samples, complete the following steps:
    On the development kit programmed with the light bulb sample, **LED 3** turns on when the light bulb joins the network.
    On the development kit programmed with the light switch sample, **LED 3** turns on when the device joins the network and **LED 4** turns on when the light switch finds a light bulb to control.
 
+Reopening the network
+   If you find that you are having issues with your devices joining the Zigbee network, it may be because the connection needs to be reopened manually.
```

Comment:
I think it would be more consistent to write `.. it may be because the network needs to be reopened...` instead if `..the connection..`

Reply:
Okay, I will make this change.